tests/Mutator/Operator/ThrowTest.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/**
+ * Copyright © 2017-2018 Maks Rafalko
+ *
+ * License: https://opensource.org/licenses/BSD-3-Clause New BSD License
+ */
+declare(strict_types=1);
+
+namespace Infection\Tests\Mutator\Operator;
+
+use Infection\Mutator\Mutator;
+use Infection\Mutator\Operator\Throw_;
+use Infection\Tests\Mutator\AbstractMutator;
+
+class ThrowTest extends AbstractMutator
+{
+    protected function getMutator(): Mutator
+    {
+        return new Throw_();
+    }
+
+    public function test_replace_break_to_continue()
```

Comment:
wrong method name. was copied from Break_ mutator :)

Reply:
I'll go ahead and apply the fix in #162, since that needs rebasing now anyways